code/game/objects/items/weapons/tanks/watertank.dm

```diff
@@ -152,8 +153,8 @@
 		loc = tank.loc
 
 /obj/item/weapon/reagent_containers/spray/mister/afterattack(obj/target, mob/user, proximity)
-	if(target.loc == loc || target == tank) //Safety check so you don't fill your mister with mutagen or something and then blast yourself in the face with it putting it away
```

Comment:
the second check isn't needed, the tank's attackby already return 1 in that case.